packages/reference/src/assets/WrappedAssetCard/FetchingWrappedAssetCard.tsx

```diff
@@ -25,61 +25,69 @@ export function FetchingWrappedAssetCard(props: FetchingWrappedAssetCardProps) {
   }, [props.assetId]);
 
   const asset = assets[props.assetId];
+  const entityKey =
```

Comment:
Not sure, I'd prefer if the store would give you some kind of result object rather than overloading this into `Enity|string|undefined`

If you want the helper you're mentioning, I think you could add it to the store as it's directly coupled to the data structure the store is exposing.